static/js/alert_words_ui.js

```diff
@@ -42,33 +42,32 @@ function update_alert_word_status(status_text, is_error) {
     $alert_word_status.show();
 }
 
-function add_alert_word(alert_word) {
-    alert_word = alert_word.trim();
+function add_alert_word(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    const alert_word = $("#add-alert-word-name").val().trim();
+
     if (alert_word === "") {
-        update_alert_word_status($t({defaultMessage: "Alert word can't be empty!"}), true);
+        ui_report.client_error(
+            $t({defaultMessage: "Alert word can't be empty!"}),
+            $("#dialog_error"),
+        );
```

Comment:
Not sure we need this now, since we disable button.

Reply:
Removed.